cwa/olin/src/lib.rs

```diff
@@ -220,6 +263,10 @@ pub mod startup {
     }
 }
 
+
+///
+/// Application Interface Binary Resource
```

Comment:
This comment is inaccurate and should probably be removed.

I'll give you the fact that the upstream [resource](https://github.com/CommonWA/cwa-spec/blob/master/ns/resource.md) definition is EXTREMELY VAGUE, but a better comment might be:

```rust
/// 
/// Resources are maybe readable, maybe writable, maybe flushable and closable streams of
/// bytes. A resource is backed by implementation in the containing runtime based on the
/// URL scheme used in the `open` call.
///
```